scratch/PCA/extract_data.py

```diff
@@ -0,0 +1,123 @@
+
+# coding: utf-8
+
+# In[61]:
+
+#!/usr/bin/env python
+
+"""
+DESCRIPTION
+
+    Script for scraping process parameters and defect parameters for PCA
+    Outfitted for use with 'process_parameters.csv'
+"""
+
+from __future__ import division
+from scipy import stats
+from matplotlib import pyplot as plt
+from mpl_toolkits.mplot3d import Axes3D
+import sys, os, glob
+import numpy as np
+import json
+
+def scrape_data(plate_list=['2'], feat_list=None, resp_var=None, stat_param=None):
+
+    # Retrieve all build features
+    all_build_data = np.genfromtxt('process_parameters.csv', delimiter=',',
```

Comment:
Since you're using these as column vectors later, it may be better to read this in as a structured type,

```
all_build_data = np.genfromtxt('process_parameters.csv', delimiter=',', dtype=True)
```

You would use that later as `all_build_data['plate']`, for example.